LandTrendr.js

```diff
@@ -1502,117 +1502,108 @@ exports.getChangeMap = function(lt, changeParams){
   if(changeParams.mag.dsnr === undefined){changeParams.mag.dsnr = false}
   
   // get the segment info
-  var segInfo = getSegmentData(lt, changeParams.index, changeParams.delta);
-  changeParams.segInfo = segInfo;
-
-
-  // sort by dist type
-  var sortByThis;
-  switch (changeParams.sort.toLowerCase()){
-    case 'greatest':
-      sortByThis = changeParams.segInfo.arraySlice(0,4,5).multiply(-1); // need to flip the delta here, since arraySort is working by ascending order
-      break;
-    case 'least':
-      sortByThis = changeParams.segInfo.arraySlice(0,4,5);
-      break;
-    case 'newest':
-      sortByThis = changeParams.segInfo.arraySlice(0,0,1).multiply(-1); // need to flip the delta here, since arraySort is working by ascending order   
-      break;
-    case 'oldest':
-      sortByThis = changeParams.segInfo.arraySlice(0,0,1);
-      break;
-    case 'fastest':
-      sortByThis = changeParams.segInfo.arraySlice(0,5,6);
-      break;
-    case 'slowest':
-      sortByThis = changeParams.segInfo.arraySlice(0,5,6).multiply(-1); // need to flip the delta here, since arraySort is working by ascending order
-      break;
-  }
+  var segInfo = ltgee.getSegmentData(lt, changeParams.index, changeParams.delta);
```

Comment:
I'm confused why we would be calling ltgee here --isn't that this file?  Why do we need the ltgee prefix?

Reply:
Good catch, thanks!
Yes, `ltgee` should not be called, this was left in by mistake when I transferred the changes into this file from an external development file.
I've removed it.